roles/openshift_health_checker/openshift_checks/package_version.py

```diff
@@ -0,0 +1,25 @@
+from openshift_checks import OpenShiftCheck, OpenShiftCheckException
+from openshift_checks.mixins import NotContainerized
+
+
+class PackageVersion(NotContainerized, OpenShiftCheck):
+    """Check that available RPM packages match the required versions."""
+
+    name = "package_version"
+    tags = ["preflight"]
+
+    @classmethod
+    def is_active(cls, task_vars):
+        return (
+            super(PackageVersion, cls).is_active(task_vars)
+            and task_vars.get("deployment_type") == "openshift-enterprise"
```

Comment:
It might be good to surface up deployment_type up to the base class as well, then you could have checks that are specific to a certain type of deployment_type and is_active could take it into account automatically as well.